investigation-US-election-2016.Rmd

```diff
@@ -0,0 +1,254 @@
+---
+title: "investigation-US-election-2016"
+output: pdf_document
+---
+
+```{r knitropts, include = FALSE, echo = FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+# knitr::opts_chunk$set(tidy.opts = list(width.cutoff = 80), tidy = TRUE)
+```
+```{r libs, message = FALSE}
+library(conflicted)
+	conflicts_prefer(dplyr::filter, dplyr::lag)
+library(tidyverse)
+library(R6)
+library(formatR)
+```
+```{r readpolls, include = FALSE}
+polls = readr::read_delim("/home/mitch/courses/data-analysis/datasets/polls_us_election_2016.csv")
+```
+```{r usepolls}
+allstates = polls$state |>
+	unique() |>
+	sort()
+polls = polls |>
+	mutate(
+		interval = startdate %--% enddate,
```

Comment:
"%--%" not defined. Likely missing package.

Reply:
right, this is from `lubridate`. I think restoring the renv should be what you need

Reply:
I added lubridate as a library in the preamble